dogs.css

```diff
@@ -0,0 +1,78 @@
+body {
+  font-family: 'Josefin Slab', serif;
+  font-size: 1.2rem;
+}
+
+/* Header */
+header {
+  background-image: url(images/header-image.jpg);
+}
+
+header img {
+  max-width: 35%;
+  height: auto;
+  /* float: left; */
+}
+
+header p {
+  color: green;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: bold;
+}
+
+.checkout-page-link {
+  color: white;
+  float: right;
+}
+
+.menu-bar {
+  background: lightgrey;
+  padding: 1px;
+}
+
+li {
+  display: inline;
+}
+
+a {
+  color: darkblue;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+h2 {
+  margin-left: 1rem;
+}
+
+.all-dogs-box {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.dog-box-wrapper {
+  flex: 1 0 21%;
+  margin: 1rem;
+}
+
+.dog-box-wrapper img {
+  width: 100%;
+  min-width: 200px;
```

Comment:
this min-width might be better used on the dog tile container itself, not the image